e2e/create-cluster_test.go

```diff
@@ -726,7 +726,7 @@ var _ = Describe("CreateCluster", func() {
 
 	})
 
-	It("creates a simple cluster with persistent VMs", Label("persistentVMs"), func() {
+	FIt("creates a simple cluster with persistent VMs", Label("persistentVMs"), func() {
```

Comment:
typo?

Reply:
Ah, i didn't mean to leave that in. Adding the `F` causes that single test to run. It's a test `focus`. I'll fix that real quick

Reply:
neat!